uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -169,6 +166,33 @@ define([
       });
     });
 
+    describe('balanceChangeByAbsenceType() with error from server', function () {
```

Comment:
the spy definition changes with main test block/suite and was giving error that its already being spied upon, hence kept it in separate suite